src/main/java/pl/javastart/task/VehicleQueue.java

```diff
@@ -0,0 +1,46 @@
+package pl.javastart.task;
+
+import java.io.*;
+import java.util.LinkedList;
+import java.util.Scanner;
+
+public class VehicleQueue extends LinkedList<Vehicle> {
+    public void makeFile(String fileName) {
+        if (isEmpty()) {
+            return;
+        }
```

Comment:
Jeżeli kolejka będzie pusta, to powinieneś wyczyścić plik, żeby pozbyć się z niego już obsłużonych pojazdów